src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -2392,8 +2433,10 @@ simpTable envname hasWidthParameter = try $ do
   spaces
   skipMany hline
   spaces
-  header' <- option [] $ try (parseTableRow envname prefsufs <*
+  header' <- option (Row nullAttr []) $ try (parseTableRow envname prefsufs <*
```

Comment:
This is because the function `parseTableRow` returns a `Row` now, rather than `[Block]`. It was much simpler to have this behavior, than to have some extra `[Block] -> Row` function involved.

Reply:
I was thinking along the lines of

``` haskell
  header' <- option [] . try . fmap (:[]) $
             parseTableRow envname prefsufs <* lbreak <* many1 hline
```